backend/models/material.model.ts

```diff
@@ -0,0 +1,34 @@
+import mongoose, { Document, Schema } from "mongoose";
+
+// const Currency = {
+// 	UkCurrency: "GBP",
+// 	UsCurrency: "USD",
+// } as const;
+
+// type CurrencyType = typeof Currency[keyof typeof Currency];
+
+// export type IMaterial = {
+// 	// supplierId: String;
+// 	name: String;
+// 	price: Number;
+// 	// currency: CurrencyType;
+// 	description?: String;
+// };
+
+// export interface MaterialModel extends IMaterial, Document {}
+
+const materialSchema = new Schema({
+	// supplierId: { type: String, required: true },
```

Comment:
You would probably need to add this back in at some point once your ready to introduce suppliers - but I support keeping it simple for now